external/fv3fit/fv3fit/_shared/models.py

```diff
@@ -232,7 +232,7 @@ def dump(self, path: str):
     def load(cls, path: str) -> "TransformedPredictor":
         with fsspec.open(os.path.join(path, cls._CONFIG_FILENAME), "r") as f:
             config = yaml.safe_load(f)
-        base_model = io.load(config["base_model"])
```

Comment:
This fixes a bug where TransformedPredictor.load can't find the base model directory if the model being loaded is not in the current working directory.